cves/kernel/CVE-2015-8787.yml

```
@@ -75,7 +80,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1300731]
```

Similar to https://github.com/VulnerabilityHistoryProject/vulnerabilities/pull/205#discussion_r1390724560

```suggestion
bugs:
  - https://bugzilla.redhat.com/show_bug.cgi?id=1300731
```